lib/rollbax/logger.ex

```diff
@@ -73,69 +20,24 @@ defmodule Rollbax.Logger do
     {:ok, state}
   end
 
-  def handle_event({level, _gl, event}, %{metadata: keys, blacklist: blacklist} = state) do
-    if proceed?(event) and meet_level?(level, state.level) do
-      post_event_unless_blacklisted(level, event, keys, blacklist)
-    end
-    {:ok, state}
-  end
+  def handle_event({:error_report, _gl, {_pid, :crash_report, [crash_info | _rest]}}, state) do
+    {{kind, {value, stacktrace}, _stacktrace}, process_info} = Keyword.pop(crash_info, :error_info)
 
-  # TODO: in the future, we probably need to flush all responses from Rollbar's
-  # API coming from logged messages, so nothing is printed.
-  def handle_event(:flush, state) do
-    {:ok, state}
-  end
+    {kind, value} =
+      if kind == :exit and Exception.exception?(value) do
+        {:error, value}
```

Comment:
I wonder: does this rewriting make sense? Leaving `kind` as `:exit` doesn't play well with Rollbar since they show it as `exit: %BadMapError{...}` (cause we ofc don't call `Exception.message/1`), but by rewriting to `:error` while we do get a nice message (`BadMapError: ...`), we don't tell the user that this was an exit. Thoughts?